server/services/rflink/api/rflink.parse.RFtoObject.js

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable prefer-destructuring */
+// eslint-disable-next-line jsdoc/check-alignment
+/**
+ * @description convert a rflink message to an array.
+ * @param {string} data - Secure node.
+ * @returns {Object} Return an array with an index for each value.
+ * @example
+ * rflink.RftoObj(data);
+ */
+function RfToObj(data) {
+  const newDevice = {};
+  const msg = String(data).split(';');
+
+  if (parseInt(msg[0], 10) === 20 && parseInt(msg[1], 10) !== 0 && msg[2] !== 'OK' && msg[3] !== undefined) {
```

Comment:
For you information,
you can

```javascript
const [part1, part2, part3, part4] = array.split(';');
```
instead of `msg[0] msg[1]....`

--- 

I fell anxious with the parseInt that can throw an exception if NaN.
Don't you prefer to compare strings?

---

I think `msg[3] !== undefined` will throw an IndexOutOfBound exception if index 3 no exist, am I wrong?